internal/printer/printer.go

```diff
@@ -3953,7 +3953,7 @@ func (p *Printer) emitStatement(node *ast.Statement) {
 		p.emitNamespaceExportDeclaration(node.AsNamespaceExportDeclaration())
 	case ast.KindImportEqualsDeclaration:
 		p.emitImportEqualsDeclaration(node.AsImportEqualsDeclaration())
-	case ast.KindImportDeclaration:
```

Comment:
Printing `KindJSImportDeclaration` should either panic (if you expect it to always be transformed), or have its own distinct emit.